### ISSUE TITLE
Move source of truth for EVMConnect/FFTM blockchain transaction status to from Core to FFTM

### ISSUE THREAD

**Issue by peterbroadhurst** [hyperledger/firefly#1108]
In https://github.com/hyperledger/firefly/issues/989 we had an observation as follows:

> We are now spending a lot of time processing receipts, and updating operations, with how chatty EVMConnect is. Large payloads, more often. Optimizing this would be best achieved by switching the webhooks from EVMconnect to just be status changes, and making the large payload be queried through on demand (as we made changes like this on listeners for example).

There is another challenge with the fact that we do best-effort WebSocket updates of operations from EVMConnect -> FF Core:
- If a websocket event is missed that changes a tranasction from Pending -> Succeeded, we will never reconcile that state

I propose a straight forward change that is aligned with a number of recent tweaks (specifically in #1028 ):
- We introduce a `status` section to `operation` 
- We only send a WebSocket updates from FFTM (EVMConnect) to Core when they result in a state transition:
    - `Pending`->`Succeeded`/`Failed`
- We only include a small `output` body in these WebSocket transactions
    - Transaction hash
    - Blockchain identifier of the form `0000012345/0000032/0000001`
- We add a `fetchstatus` option to the `GET` `/operations/{operationId}` calls\
    - Makes an in-line `GET` to FFTM/EVMConnect for the latest full body of the `/transactions/{txId}` FFTM API
    - This is returned under a `status` field in the response to the `/operations/{operationId}` call
    - The UI enables `fetchstatus` by default
- In the case that the status of the backend during such a `GET` is resolved to `Succeeded`/`Failed`, but the status of the operation is still `Pending` - we dispatch an update of the FF Core status, as if we'd just received the websocket event

**Comment by matthew1001** [comment]
I've made the majority of the changes for this task, currently under a branch in https://github.com/kaleido-io/firefly/tree/fetch-status-changes 

There are a couple of details before I submit a PR:

- Operation already has a `status` field so I've coded it with a new `detailedStatus` for now
- The UI defaults to showing `Input` and `Output` for an operation. Switching the UI to use `fetchstatus` will return the extra data but I'm just deciding whether to add `DetailedStatus` as an extra section for `Operations`

**Comment by matthew1001** [comment]
Using the `detailedStatus` section described above and with the UI changed to use `fetchstatus=true` the operation slideout panel now looks like this:

![image](https://user-images.githubusercontent.com/6599269/204341425-a087b06c-aa62-4820-b0f7-5d45c682fc55.png)

**Comment by matthew1001** [comment]
UI updates currently in https://github.com/kaleido-io/firefly-ui/tree/fetch-detailed-operation-status

**Comment by peterbroadhurst** [comment]
@matthew1001 - I think we should merge https://github.com/hyperledger/firefly-transaction-manager/compare/main...kaleido-io:firefly-transaction-manager:fetch-status#diff-718d2f38d7576dfd1ff533a84c2be68b4660be6ed0fc60f6e50af9097395141cR298 with https://github.com/hyperledger/firefly-transaction-manager/pull/43

I also note the protocolID needs to include the transaction index with the formatting we use for event signatures, per this function I added in PR 43: https://github.com/hyperledger/firefly-transaction-manager/blob/fb62150620c3008d9749a427a04961353be51e5f/pkg/ffcapi/api.go#L137-L142

**Comment by matthew1001** [comment]
Yes agreed. Sorting out protocol ID was the last thing on my list to do so I'll get that sorted.
